lib/cs_guide/categories/drink_type.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CsGuide.Categories.DrinkType do
+  use Ecto.Schema
+  use Alog
+  import Ecto.Changeset
+
+  schema "drink_types" do
+    field(:name, :string)
+    field(:entry_id, :string)
+    field(:deleted, :boolean)
+
+    many_to_many(
+      :drinks,
+      CsGuide.Resources.Drink,
+      join_through: "drink_drink_types",
+      join_keys: [drink_type_id: :id, drink_id: :id]
+    )
+
+    timestamps()
```

Comment:
we probably need to over-ride this phoenix macro as we don't need the `updated_at` column.